lib/stdlib/fields.juttle

```diff
@@ -0,0 +1,74 @@
+// Manipulate fields within a stream with these field reducers
+
+// Return an array of last N values of a field
+//
+// Parameters:
+//
+//   * field: ("value") field to tail
+//   * length: (1) number of recent values to retain
+export reducer tail(field, length=1) {
+    var data = [];
+
+    function update() {
+        if (*field != null) { 
```

Comment:
Don't have a strong preference, but my first guess would've been that `tail` _does not_ ignore `null` values. Fine with it either way.